### PR TITLE
fix hugo base url

### PR DESCRIPTION
**Pull request by nicolastakashi**
adding small fixed that makes the website work as expeceted.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,19 +1,20 @@
-version: '3.9'
+version: "3.3"
```
Comment (Nexucis) [inline docker-compose.yaml:1]: oh do we really need to downgrade the schema version ?

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-      args:
-        - HUGO_BASEURL=https://perses.dev
+    image: hugomods/hugo
+    command: hugo server --watch --bind=0.0.0.0 --disableLiveReload=true --minify -environment production --baseURL https://perses.dev --appendPort=false
     restart: unless-stopped
     ports:
-      - "5001:80"
+      - 1313:1313
+    volumes:
+      - ./website:/src
```
Comment (Nexucis) [inline docker-compose.yaml:11]: does that mean we need to run on the server the script `synchronize.sh` ? And if so we should probably add to ignore file every file generated if it's not done already

File: docker-compose.yaml (hunk 1 above)
Comment (nicolastakashi) [reply docker-compose.yaml:1]: Oh!
Now that's what by mistake, going to fix it.

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
     labels:
       - "traefik.enable=true"
       # The domain the service will respond to
       - "traefik.http.routers.website.rule=Host(`perses.dev`)"
       - "traefik.http.routers.website.tls.certresolver=myresolver"
+      - "traefik.http.services.website.loadbalancer.server.port=1313"
```
Comment (Nexucis) [inline docker-compose.yaml:17]: that's curious we need to add this label. I thought traefic were able to find using the docker discovery what was the port to use to send the traefic. But maybe I'm wrong

File: docker-compose.yaml (hunk 2 above)
Comment (nicolastakashi) [reply docker-compose.yaml:11]: I'll double check it, but I guess I already ignore it.

Comment (Nexucis) [reply docker-compose.yaml:11]: based on what you did directly on the server, it looks like it's not ignored : 

```bash
$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   docker-compose.yaml
	modified:   scripts/synchronize.sh

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	docker-compose.old
	package-lock.json
	website/.hugo_build.lock
	website/content/docs/perses/
	website/content/docs/prologue/
	website/resources/_gen/assets/
	website/static/images/
```

File: docker-compose.yaml (hunk 3 above)
Comment (nicolastakashi) [reply docker-compose.yaml:17]: Actually I didn't tried 😄
I can check if it's really required

Comment (nicolastakashi) [reply docker-compose.yaml]: @Nexucis it works without this one.

File: docker-compose.yaml (hunk 2 above)
Comment (nicolastakashi) [reply docker-compose.yaml:11]: @Nexucis added can you check?

**Review by Nexucis — APPROVED** [review]
so cool you managed to make it work !

Thank you so much 🙏

**Pull request merged**